### ISSUE TITLE
wirehair failures

### ISSUE THREAD

**Issue by smanders** [smanders/externpro#334]
wirehair was added to externpro 21.01
* add wirehair https://github.com/smanders/externpro/issues/291
* add cm256 https://github.com/smanders/externpro/issues/292

wirehair built as part of externpro fails (executable using wirehair library crashes, but only on one specific system) -- a wirehair patch was made to attempt to remove AVX2 dependencies from the code
* wirehair patch https://github.com/smanders/externpro/issues/320

the executable using the library still crashes

commits
* wirehair patch https://github.com/catid/wirehair/compare/9e8923a200045c7bdb9e89b449692dbfed6b6de0...smanders:xp20.11.21
* patch w/o removing AVX2 dependencies https://github.com/catid/wirehair/compare/9e8923a200045c7bdb9e89b449692dbfed6b6de0...26f0e120032b80af027ae322a681a0bea10240e7

**Comment by smanders** [comment]
failure/issue is reproducible

copying the unit_test executable from the build system to the runtime system (which fails)
```
scp xpbase/Build/wirehair_Release/unit_test root@172.31.67.25:~
```

and then running it on the 172.31.67.25 system
```
[root@dde-dev1 ~]# ./unit_test
Illegal instruction
```

**Comment by smanders** [comment]
there have been a number of commits/fixes upstream since the 2020.11.21 commit the current build is based on - up to the current HEAD of master (a 2021.07.31 commit) https://github.com/catid/wirehair/compare/9e8923a200045c7bdb9e89b449692dbfed6b6de0...6d84fad40cbbbb29d4eb91204750ddffe0dcacfe

one commit in particular provides the "fix" - an option to *not* build with `-march=native` https://github.com/catid/wirehair/commit/6d84fad40cbbbb29d4eb91204750ddffe0dcacfe

**Comment by smanders** [comment]
21.07.31 patch https://github.com/catid/wirehair/compare/6d84fad40cbbbb29d4eb91204750ddffe0dcacfe...smanders:xp21.07.31

completed with commit referenced above
used in dde and dde_lib
